gnosis/safe/addresses.py

```diff
@@ -181,6 +181,10 @@
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 11545, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 11546, "1.3.0"),
     ],
+    EthereumNetwork.ARBITRUM_SEPOLIA: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 154, "1.3.0+L2"),
+        ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 155, "1.3.0"),
+    ],
```

Comment:
Please add also ProxyFactory address and block on PROXY_FACTORIES section https://github.com/safe-global/safe-eth-py/pull/669/files#diff-62bfa04ec3dcf06fa5a67465980b383d49368b25afe14cc3abb576b003d25062R604

Reply:
Done